test/plugin/test_out_google_cloud.rb

```diff
@@ -176,6 +178,7 @@ def test_managed_vm_metadata_loading
   end
 
   def test_gce_metadata_does_not_load_when_fetch_gce_metadata_is_false
```

Comment:
This is the same as "test_custom_metadata", because setting the stubs shouldn't matter if they are never used?
Which would also mean, that you can put the stubs back into the main Setup() body?